server.lua

```diff
@@ -33,7 +33,7 @@ end
 ---@param amount integer Amount of item to be added to inventory
 local function addItem(item, amount)
     if onCooldown(20) then return end
-    local player = QBCore.Functions.GetPlayer(source)
+    local Player = QBCore.Functions.GetPlayer(source)
```

Comment:
camelCase https://manason.github.io/effective-fivem-lua/variables/#camelcase-non-constant-local-variable-names

Reply:
It was changed as it was throwing a unused variable linting error. Most of the qb serverside lua has `local Player = QBCore.Functions.GetPlayer(source)` so I was sticking with the theme.

Reply:
It's unused because the following line needs to be lower case as well.

Reply:
Just pushed another commit to fix it for the whole file